src/audit-log-api/src/handlers/sanitiseMessage.e2e.test.ts

```diff
@@ -1,6 +1,6 @@
 jest.setTimeout(15000)
 import { setEnvironmentVariables } from "shared-testing"
-setEnvironmentVariables({ BICHARD_DB_TABLE_NAME: "archive_error_list" })
+setEnvironmentVariables({ BICHARD_DB_TABLE_NAME: "br7own.archive_error_list", DB_SCHEMA: "br7own" })
```

Comment:
I wonder if we could also set `DB_SCHEMA` in `setEnvironmentVariables` function. I think it's always the same value like other environment variables in that function.

Also regarding `BICHARD_DB_TABLE_NAME` variable, I think we can remove this environment variable. It's not used anywhere.

Reply:
Fair point, done 👍